source/extensions/access_loggers/file/config.cc

```diff
@@ -61,8 +61,16 @@ FileAccessLogFactory::createAccessLogInstance(const Protobuf::Message& config,
     formatter = Formatter::SubstitutionFormatUtils::defaultSubstitutionFormatter();
     break;
   }
+  Filesystem::FilePathAndType file_info{
+      static_cast<Filesystem::DestinationType>(fal_config.access_log_destination()),
+      fal_config.path()};
 
-  return std::make_shared<FileAccessLog>(fal_config.path(), std::move(filter), std::move(formatter),
+  if (file_info.path_.empty() && file_info.file_type_ == Filesystem::DestinationType::File) {
```

Comment:
What about the case: !file_info.path_.empty() && file_info.file_type_ != Filesystem::DestinationType::File

Reply:
This is implicit behavior and not clear. It is assumed that if `Filesystem::DestinationType` is set then the `path_` property is ignored.

Should we consider this a misconfiguration and throw?

Reply:
I can see 2 possible options:
- Log to both the requested special destination and the file path specified.
- consider it a misconfiguration and throw.

I think that considering it misconfiguration is easier.

Reply:
I went with approach (2) in 74bc1cb7efffccfa229203ae7fc66c98a82f1b18